vim/merlin/autoload/neomake/ocaml.vim

```diff
@@ -0,0 +1,8 @@
+function! neomake#makers#ft#ocaml#merlin()
+  let maker = {'name': 'merlin'}
```

Comment:
I would assume this to be the default name.. but have not checked.
Try just an empty dict here.